src/STAT_FrontEnd.h

```diff
@@ -175,17 +164,14 @@ class STAT_FrontEnd
 #endif
 
     public:
-        //! Default constructor
```

Comment:
Create via a factory method, for reference the lmon implementation:
#ifndef USE_CTI
STAT_FrontEnd* STAT_FrontEnd::make()
{
    return new STAT_lmonFrontEnd();
}
#endif
Since I'm just determining the backing implementation at build time.